test/common/ops/openMysteryItem.js

```diff
@@ -36,7 +36,9 @@ describe('shared.ops.openMysteryItem', () => {
 
     expect(user.items.gear.owned[mysteryItemKey]).to.be.true;
     expect(message).to.equal(i18n.t('mysteryItemOpened'));
-    expect(data).to.eql(content.gear.flat[mysteryItemKey]);
+    let item = content.gear.flat[mysteryItemKey];
```

Comment:
hmm this changes the value of the item text for all the tests after this, we shouldn'd really mutate this object

Reply:
also I'm not sure this is necessary, I mean before it returned the english text? In that case it should work anyway as the tests should use english unless otherwise specified

Reply:
without the line 40 here, the item does not contain the name in english. instead `text` is a function reference. Which is also why the API doesn't return it, since the function was never actually called.

Not sure how I should handle this test otherwise? Should I create a copy of the item and use that?

Reply:
yeah I'd probably clone it with lodash's cloneDeep function